rust/src/bin/lq.rs

```diff
@@ -115,7 +116,19 @@ async fn main() -> Result<()> {
             scanner.limit(*n, None).unwrap();
             let stream = scanner.try_into_stream().await.unwrap();
             let batch: Vec<RecordBatch> = stream.take(1).try_collect::<Vec<_>>().await.unwrap();
-            println!("{:?}", batch);
+
+            // print the summary of data total rows and cols
+            let mut total_num_of_rows: usize = 0;
+            let mut total_num_of_cols: usize = 0;
+            for b in batch.iter() {
+                total_num_of_rows += b.num_rows();
+                total_num_of_cols += b.num_columns();
```

Comment:
The columns could be the same amount batches. We could just print num of columns from the first batch.

For the number of rows, let's wait for #495 to get in?

The rest LGTM

Reply:
done

Reply:
you are right the cols would be same from the 1st batch